pdudaemon/drivers/strategies.py

```diff
@@ -33,6 +33,7 @@
 from pdudaemon.drivers.apc9210 import APC9210
 from pdudaemon.drivers.apc7920 import APC7920
 from pdudaemon.drivers.apc7921 import APC7921
+from pdudaemon.drivers.cambrionix import CambrionixPDSyncC4
```

Comment:
merge the commit where this is in wit hthe one introducing the driver; This is not CVS :) one file per commit isn't helpful

Reply:
the CI also runs linter checks on each commit of a PR; this change on its own causes the ruff linter in CI to fail, since the `CambrionixPDSyncC4` doesn't yet exist.